client/multi-currency/index.js

```diff
@@ -51,6 +51,14 @@ if ( storeSettingsSection ) {
 	toggleSettingsSectionDisplay();
 }
 
+function realEnabledCurrenciesListItems() {
```

Comment:
Hey Taha! Good job on the PR, looks good 👍 

A small detail: Instead of `realEnabledCurrenciesListItems` for the name of the function, I think we could use something more descriptive, as I find `real` a bit abstract from a first read 😸 Because this is checking for the currency elements that are not placeholder, maybe it could be named `renderedEnabledCurrenciesListItems`, or something similar.

Does it make sense? 🤔 

Thank you! 🙌

Reply:
Yes it makes sense, I couldn't find a better name, left it as it is to return back and fix it later, and didn't return back 😅 I'll change it to a clearer one.